server/src/test/scala/ch.epfl.scala.index.server/routes/FrontPageTests.scala

```diff
@@ -0,0 +1,68 @@
+package ch.epfl.scala.index.server.routes
+
+import ch.epfl.scala.index.model.Project
+import ch.epfl.scala.index.model.release.BinaryVersion
+import ch.epfl.scala.index.model.release.PatchBinary
+import ch.epfl.scala.index.model.release.Platform
+import ch.epfl.scala.index.model.release.Scala3Version
+import ch.epfl.scala.index.model.release.ScalaLanguageVersion
+import ch.epfl.scala.index.model.release.ScalaVersion
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+
+class FrontPageTests extends AnyFunSpec with Matchers {
+
+  describe("FrontPageTests") {
+    import ScalaVersion._
+    import Scala3Version._
+    import Platform.PlatformType._
+    val js0618 = PatchBinary(0, 6, 18)
+    val nat03 = PatchBinary(0, 3, 0)
+    val nat04 = PatchBinary(0, 4, 0)
+    val ref1 = Project.Reference("a", "b")
+    val ref2 = Project.Reference("c", "d")
+    val `3.0.0-RC3` =
+      Platform.ScalaJvm(ScalaLanguageVersion.tryParse("3.0.0-RC3").get)
+    val `3.0.0-RC1` =
+      Platform.ScalaJvm(ScalaLanguageVersion.tryParse("3.0.0-RC1").get)
+    val platformWithCount = Map(
+      ref1 -> Set(
+        Platform.Java,
+        Platform.ScalaNative(`2.11`, nat03),
+        Platform.ScalaNative(`2.12`, nat03),
+        Platform.ScalaNative(`2.12`, nat04)
+      ),
+      ref2 -> Set(
+        Platform.ScalaNative(`2.13`, nat03),
+        Platform.ScalaJvm(`2.13`),
+        Platform.ScalaJvm(`2.12`),
+        Platform.Java,
+        `3.0.0-RC3`,
+        `3.0.0-RC1`
+      )
+    )
```

Comment:
All these could be put in `Values` and be used in other test classes, in `PlatformTests` or `BadgesSupportTest` for instance.